src/diffusers/models/prior_transformer.py

```diff
@@ -54,24 +71,53 @@ def __init__(
         num_embeddings=77,
         additional_embeddings=4,
         dropout: float = 0.0,
+        time_embed_act_fn: str = "silu",
+        norm_in_type: Optional[str] = None,  # layer
+        embedding_proj_norm_type: Optional[str] = None,  # layer
+        encoder_hid_proj_type: Optional[str] = "linear",  # linear
+        added_emb_type: Optional[str] = "prd",  # prd
+        time_embed_dim: Optional[int] = None,
+        embedding_proj_dim: Optional[int] = None,
+        clip_embed_dim: Optional[int] = None,
+        upcast_softmax: bool = False,
```

Comment:
```suggestion
```

I couldn't find this anywhere in the configs - do we need it?

Reply:
https://huggingface.co/openai/shap-e/blob/main/prior/config.json

Reply:
@patrickvonplaten here https://github.com/openai/shap-e/blob/main/shap_e/models/generation/transformer.py#L79